draft-ietf-taps-interface.md

```diff
@@ -2456,7 +2456,15 @@ Framers must happen before any operation that may result in the creation of a Co
 #### Framing Meta-Data {#framing-meta}
 
 When sending Messages, applications can add Framer-specific
-key/value pairs to a MessageContext ({{msg-ctx}}).
+properties to a MessageContext ({{msg-ctx}}).
+In order to set these properties, the `add` and `get` actions
+on the MessageContext take an additional framer parameter.
+
+~~~
+MessageContext.add(framer, property, value)
+PropertyValue := MessageContext.get(framer, property)
+~~~
+
```

Comment:
```suggestion
properties to a MessageContext ({{msg-ctx}}).
In order to set these properties, use the `add` and `get` actions
on the MessageContext. To avoid naming conflicts, the property names SHOULD be prefixed with a namespace referencing the framer implementation or the protocol it implements as described in {{property-names}}. 

~~~
MessageContext.add(property, value)
PropertyValue := MessageContext.get(property)
~~~

```

Reply:
```suggestion
properties to a MessageContext ({{msg-ctx}}).
In order to set these properties, the `add` and `get` actions
on the MessageContext. To avoid naming conflicts, the property
names SHOULD be prefixed with a namespace referencing the
framer implementation or the protocol it implements as described
in {{property-names}}. 

```